apps/web/src/app/challenge/_components/challenge-layout.tsx

```diff
@@ -161,15 +165,19 @@ export function ChallengeLayout({ left, right }: ChallengeLayoutProps) {
       ref={parent}
       style={{ height: 'calc(100dvh - 3.5rem)' }}
     >
-      <div
-        className="min-h-[318px] w-full overflow-hidden rounded-l-2xl rounded-r-xl border border-zinc-300 bg-white dark:border-zinc-700 dark:bg-zinc-800 lg:min-w-[500px]"
-        ref={leftSide}
-      >
-        {left}
-      </div>
-      <div className="resizer relative p-2" ref={resizer}>
-        <div className="absolute left-1/2 top-1/2 h-1 w-24 -translate-x-1/2 -translate-y-1/2 rounded-full bg-neutral-400 duration-300 group-hover:bg-neutral-600 group-active:bg-emerald-400 group-active:duration-75 dark:bg-neutral-700 group-hover:dark:bg-neutral-500 lg:h-24 lg:w-1" />
-      </div>
+      {!settings.isFullscreen && (
```

Comment:
Maybe we should reduce the width of leftpanel to 0 instead and keep the resize handle as this setting persists across page loads and not indicate visually that left panel is hidden

Reply:
I'm fine with it either way had a couple q's out in discord on the preferred method.